tests/dummy/app/helpers/i.js

```diff
@@ -0,0 +1,9 @@
+import Ember from 'ember';
+
+const matcher = /\$\{\s*(.*?)\s*\}/g;
+
+export function interpolate([ str ], hash) {
```

Comment:
yes, I think we should move the demo to ember-sparkles and remove this file (and maybe also the dependency you added in `package.json` ? )